spec/spec_helper.rb

```diff
@@ -12,12 +12,11 @@
 
 Time.zone_default = Time.find_zone!('UTC')
 
-ActiveRecord::Base.configurations['test'] = {
-  'adapter'  => 'sqlite3',
-  'database' => Tempfile.new(['fleiss-test', '.sqlite3']).path,
-  'pool'     => 20,
-}
+database_url = ENV['DATABASE_URL'] || "sqlite3://#{Tempfile.new(['fleiss-test', '.sqlite3']).path}"
+ActiveRecord::Base.configurations['test'] = { 'url' => database_url, 'pool' => 20 }
```

Comment:
I have a feeling that sqlite even with a huge pool is not the best database for concurrency testing (can be mistaking it for something else, but I remember like long queries/transactions against single sqlite db were "blocking" each other when being run in `rails s` and `rails c` concurrently).